### ISSUE TITLE
I would like to translate Vietnamese version of wpHierarchy

### ISSUE THREAD

**Issue by dtvn** [pressupinc/wphierarchy#34]
Is it ok to do that? If so, where should I start?

**Comment by davidbhayes** [comment]
Good question! I think some internationalization work exists on the site, but I've not looked into it. 

What are you looking to translate? Essentially the gray-boxes on the diagram? Or would other things also make sense to change?

**Comment by dtvn** [comment]
Yup, gray-boxes on the diagram, the menu, and further is the article. It is a very good resource that I think it could be more helpful to Vietnamese people if it could be translated.
